Lib/test/test_subprocess.py

```diff
@@ -1378,6 +1378,29 @@ def test_file_not_found_with_bad_cwd(self):
             subprocess.Popen(['exit', '0'], cwd='/some/nonexistent/directory')
         self.assertEqual(c.exception.filename, '/some/nonexistent/directory')
 
+    @support.skip_unless_symlink
+    def test_defpath(self):
+        # bpo-35755: Test that Python doesn't run a program from the current
+        # directory when the PATH environment variable is not set.
+        # Test directly the value of os.defpath.
+        program = 'subprocess_test_defpath.exe'
+        args = (program, '-I', '-S', '-c', 'pass')
+
+        env = dict(os.environ)
+        env.pop('PATH', None)
+
+        with support.temp_cwd() as temp_dir:
+            self.addCleanup(support.rmtree, temp_dir)
```

Comment:
Oh. It's now useless, I forgot to remove it. I will remove it once I understand the Windows failure.